types/prettier/prettier-tests.ts

```diff
@@ -226,7 +226,7 @@ const plugin: prettier.Plugin<PluginAST> = {
             since: '1.0.0',
             type: 'path',
             category: 'Test',
-        }
+        },
     },
 };
 
```

Comment:
A new test should be added.

Reply:
@JounQin thanks for the feedback but I don't know how I can test this change. I've been reviewing the current test and I can't understand them because it seems that they are testing formats but not current options.

Reply:
Adding `prettier.format('another', { bracketSameLine: true });` to the bottom of `prettier.test.ts` should be sufficient to test this change.